### DISCUSSION TITLE
Modern Neverball icons

### DISCUSSION THREAD

**Discussion by maxtron95** [Neverball/neverball#339] [Show and tell]
I decided to make modern redesigns of the Neverball and Neverputt icons in svg, png, and ico formats.

![neverball-modern-preview-wip](https://github.com/Neverball/neverball/assets/53346722/88cd3e5d-8847-420c-a300-9ec1588797de)

[Neverball.zip](https://github.com/Neverball/neverball/files/12187758/Neverball.zip)

**Comment by parasti** [comment]
@creepertron95 These are cool! Do you mind if I add them to the Neverball repo? Are they licensed in a manner that's compatible with GPL?

**Comment by maxtron95** [comment]
You can add them to the repo. I had designed these as updated replacements to the original icons. I didn't have a license in mind when I originally made these so it should be fine to license them however you want.

**Comment by parasti** [comment]
@creepertron95 Awesome!

I'd also like to add you to this list: https://github.com/Neverball/neverball/blob/master/doc/authors.txt

Let me know how if you want to be credited a certain way.

**Reply by maxtron95** [reply]
@parasti I have recently updated my username to @maxtron95.
